mvg/plotting.py

```diff
@@ -222,23 +234,21 @@ def modes_over_time(
 
     ts_range = data["timestamps"].iloc[-1] - data["timestamps"].iloc[0]
     scaling_factor = len(data) / ts_range
+    # List of (y_pos, height) of each row we want to plot
+    row_pos_height = [(0, 0.85 * height), (0.85 * height, 0.2 * height)]
```

Comment:
Better convert this into a dictionary. It is very hard to understand the purpose of these values when they are used like `row_pos_height[1][0]`. My understanding is that the first element is for the modes and the second element is for uncertain areas. It is not clear fro the code or comments.